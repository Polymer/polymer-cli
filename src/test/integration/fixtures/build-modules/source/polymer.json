{
  "npm": true,
  "moduleResolution": "node",
  "builds": [
    {
      "name": "unbundled-es",
      "bundle": false
    },
    {
      "name": "unbundled-amd",
      "bundle": false,
      "js": {
        "transformModulesToAmd": true
      }
    },
    {
      "name": "bundled-es",
      "bundle": {
        "inlineScripts": false
      }
    },
    {
      "name": "bundled-amd",
      "bundle": {
        "inlineScripts": false
      },
      "js": {
        "transformModulesToAmd": true
      }
    }
  ]
}
