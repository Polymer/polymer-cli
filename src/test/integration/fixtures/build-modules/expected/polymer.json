{
  "entrypoint": "index.html",
  "fragments": [],
  "sources": [
    "src/**/*",
    "index.html"
  ],
  "extraDependencies": [
    "bower_components/webcomponentsjs/*.js"
  ],
  "builds": [
    {
      "name": "foo"
    }
  ],
  "npm": true,
  "componentDir": "node_modules/",
  "moduleResolution": "node"
}
